_pkgdown.yml

```diff
@@ -1 +1,31 @@
 destination: docs
+
+reference:
```

Comment:
What is this for? Also please add info to the Hub.

Reply:
It organizes the references page so that instead of all functions being listed in alphabetical order, you can include just the functions you want in different groups